deeds-dapp-webapp/src/main/webapp/vue-app/dapp/staticPage/components/page/Home.vue

```diff
@@ -151,7 +155,11 @@
       </div>
     </v-card>
     <div class="pt-16 mt-6">
-      <div class="v-card__title full-height justify-center display-3 font-weight-bold">{{ $t('poweredByWeb3.title') }}</div>
+      <span
+        :class="isMobile && 'display-2' || 'display-3'"
+        class="d-flex justify-center text-center font-weight-bold">
+        {{ $t('poweredByWeb3.title') }}
+      </span>
```

Comment:
please avoid using JS for Mobile checking. Use instead `text-${breakpoint}-h${X}`